tests/conf.js

```diff
@@ -93,38 +117,46 @@ const runAndLog = (msg, func) => {
   return func();
 };
 
-const prepServices = () => {
+const prepServices = async () => {
   let apiReady;
   if (constants.IS_TRAVIS) {
     console.log('On travis, waiting for horti to first boot api');
     // Travis' horti will be installing and then deploying api and sentinel, and those logs are
     // getting pushed into horti.log Once horti has bootstrapped we want to restart everything so
     // that the service processes get restarted with their logs separated and pointing to the
     // correct logs for testing
-    apiReady = listenForApi()
-      .then(() => console.log('Horti booted API, rebooting under our logging structure'))
-      .then(() => request.post('http://localhost:31337/all/restart'));
+    await listenForApi();
+    console.log('Horti booted API, rebooting under our logging structure');
+    apiReady = await request.post('http://localhost:31337/all/restart');
   } else {
     // Locally we just need to start them and can do so straight away
-    apiReady = request.post('http://localhost:31337/all/start');
+    apiReady = await request.post('http://localhost:31337/all/start');
   }
 
-  return apiReady
-    .then(() => listenForApi())
-    .then(() => runAndLog('Settings setup', setupSettings))
-    .then(() => runAndLog('User contact doc setup', utils.setUserContactDoc));
+  await listenForApi();
+  await runAndLog('Settings setup', setupSettings);
+  await runAndLog('User contact doc setup', utils.setUserContactDoc);
+  return apiReady;
+};
+
+const apiRetry = () => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(listenForApi());
+    }, 1000);
+  });
 };
 
-const listenForApi = () => {
+const listenForApi = async () => {
   console.log('Checking API');
-  return utils.request({ path: '/api/info' }).catch(() => {
+  try {
+    const result =  await utils.request({ path: '/api/info' });
+    return result;
```

Comment:
You don't need the assignment or the return statement here (because we don't need the return value from that endpoint anywhere). 

```suggestion
    await utils.request({ path: '/api/info' });
```